.appveyor.yml

```diff
@@ -0,0 +1,26 @@
+os: Visual Studio 2015
+
+environment:
+  matrix:
+  - BUILD_ENV: msbuild
+    arch: x64
+  - BUILD_ENV: msbuild
+    arch: Win32
+  - BUILD_ENV: cygwin
+
+install:
+  - cmd: if %BUILD_ENV%==cygwin (
+        C:\cygwin64\setup-x86_64.exe --quiet-mode --no-shortcuts --upgrade-also --packages autoconf,automake )
+
+build_script:
+  - cmd: if %BUILD_ENV%==msbuild (
+        msbuild .\windows\hidapi.sln /p:Configuration=Release /p:Platform=%arch% /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" )
+  - cmd: if %BUILD_ENV%==cygwin (
+        C:\cygwin64\bin\bash -exlc "cd $APPVEYOR_BUILD_FOLDER; ./bootstrap; ./configure; make" )
+
+artifacts:
+  - path: .\windows\Release\hidapi.dll
+  - path: .\windows\Release\hidapi.lib
+  - path: .\windows\Release\hidapi.pdb
```

Comment:
do we have `.pdb` files for `cygwin` build?
or is it silently ignores its absence?

Reply:
Non-existing artifacts get ignored by appveyor